src/nvim/lua/vim.lua

```diff
@@ -424,6 +424,14 @@ function vim.notify(msg, log_level, _opts)
   end
 end
 
+vim.ui = {
+  --- Picker
+  --- without a runtime, relies on inputlist()
+  --  see :help nvim_pick
+  pick = function (entries, opts)
```

Comment:
For opts I'm thinking we can have a `multiselect` option? But i'm not sure how to impl with inputlist..

Reply:
agreed. I see this as part of the next breaking change update of the feature. For now I just would like to merge a basic one with "prompt" in options.

Reply:
Then this should be `pick_one` IMO